src/main/java/de/unistuttgart/bugfinder/BugfinderServiceApplication.java

```diff
@@ -0,0 +1,14 @@
+package de.unistuttgart.bugfinder;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cloud.openfeign.EnableFeignClients;
+
+@SpringBootApplication(scanBasePackages = { "de.unistuttgart" })
```

Comment:
```suggestion
@SpringBootApplication
```
Becomes obsolete once `authentication-validator@v0.0.2` is released